src/screens/conclusion/index.js

```diff
@@ -44,43 +41,71 @@ export function Conclusion() {
+        <img
+          className="not-sr-only absolute right-1/4 top-[-230px] max-w-lg lg:w-full"
+          src="/images/conclusion/pups.png"
+          alt="Bear pups"
+        />
+        <div className="flex">
+          <HealthcareIcon />
+          <div className="ml-8 flex-col">
+            <h5 className="mb-2 text-xl font-bold text-white">
+              Please consider supporting our work!
+            </h5>
+            <span className="text-md font-light text-white">
               This website would not be possible without your support. Thank
               you!
-            </div>
+            </span>
           </div>
         </div>
-        <span className="uppercase">Support Us</span>
+        <a
+          href={links.veganHacktivistsSupport}
+          target="_blank"
+          rel="noreferrer"
+          className="self-center rounded-xl bg-primary px-5 py-1 text-xl font-medium uppercase"
+        >
+          Support Us
+        </a>
       </div>
-      <div className="flex flex-row">
-        <h5 className="px-20">Credits and sources</h5>
-        <div>
-          <p>
-            Mauris convallis arcu amet, sollicitudin gravida gravida platea.
-          </p>
-          <p>
-            Velit rhoncus pellentesque feugiat aenean tincidunt. Diam tincidunt
-            aliquet mauris lorem.
-          </p>
-          <p>
-            Egestas scelerisque ac risus. Neque nibh tellus elementum vulputate
-            auctor
-          </p>
-          <p>Arcu amet, sollicitudin gravida gravida platea.</p>
-          <p>Rrhoncus pellentesque feugiat aenean tincidunt</p>
-          <p>
-            Diam tincidunt aliquet mauris lorem egestas scelerisque ac risus
-          </p>
-          <p>Neque nibh tellus elementum vulputate auctor.</p>
-          <p>
-            Mauris convallis arcu amet, sollicitudin gravida gravida platea.
-          </p>
+
+      <div className="mx-auto flex max-w-screen-2xl justify-evenly p-14">
+        <div className="mr-20 flex flex-col">
+          <div className="flex flex-row text-white">
+            <h5 className="mr-16 w-[170px] min-w-[170px] font-bold">
+              Special thanks
+            </h5>
+            <div className="max-w-3xl leading-relaxed">
+              To our partners and advisors from Wild Animal Initiative, Animal
+              Ethics, Rethinking Priorities, and Jack Hancock for his
+              collaborative role. Kate Rodman for the wonderful illustrations.
+              This website would not have been possible without everyone’s
+              dedication and support!
+            </div>
+          </div>
+          <div className="mt-10 flex flex-row text-white">
+            <h5 className="mr-16 w-[170px] min-w-[170px] font-bold">
+              Credits and sources
+            </h5>
+            <div className="flex flex-col leading-relaxed">
+              {sources.map((source, i) => (
+                <Source
+                  // @ts-ignore
```

Comment:
This is annoying 🙃.  Adding `key` here results in the error below. But not including `key` results in "missing key" error. And `key` shouldn't need to be added to the props for the component. The error goes away if we turn off "checkJs` in jsconfig. This is happening other places too where we have key.
<img width="769" alt="key error" src="https://user-images.githubusercontent.com/39889198/202218321-fc304021-2895-4450-9f13-29ea199baee7.png">